schema-registry/client/src/main/java/com/hortonworks/registries/schemaregistry/client/SchemaRegistryClient.java

```diff
@@ -228,6 +245,27 @@ public SchemaVersionInfo retrieveSchemaVersion(SchemaIdVersion key) throws Schem
                 .build();
     }
 
+    protected SSLContext createSSLContext(Map<String, String> sslConfigurations) {
+        String defaultPropertyValue = null;
+        SslConfigurator sslConfigurator = SslConfigurator.newInstance();
+        sslConfigurator.keyStoreType(sslConfigurations.getOrDefault("keyStoreType",defaultPropertyValue))
+                .keyStoreFile(sslConfigurations.getOrDefault("keyStorePath",defaultPropertyValue))
```

Comment:
if any of these properties is set to null, then the SSLConfigurator would use a default value for the property.